desktop/cmp/grouping/GroupingChooser.ts

```diff
@@ -39,6 +40,126 @@ export interface GroupingChooserProps extends ButtonProps<GroupingChooserModel>
     styleButtonAsInput?: boolean;
 }
 
+class GroupingChooserLocalModel extends HoistModel {
```

Comment:
you can put it in an /impl directory, and mark it as @internal.  We do this all the time!

Reply:
Thanks for the tip, I knew we must have a pattern for this (and do this all the time in apps). Was hoping for something that would hide the class to app code, doesn't seem like there is any enforcement of `@internal` . Looks like there are lots of proposals out there for this kind of thing, hopefully something makes it in eventually.

Looks like there is a way to strip these from d.ts output - https://www.typescriptlang.org/tsconfig#stripInternal - though since we are full typescript I don't think this does anything.